vigetlabs/wordpress-site-starter
# [#6] Adding style for icon cards

# Summary

Styling the cards. This was pretty simple, but we may want to adjust this per client as I could see the icons needing a background color with an SVG. Left it simple for now. 

## Issues

* #6

## Testing Instructions

1. Add multiple cards to the component. 
2. Make sure they all flow and move to two cols on tablet and one on mobile. 

## Screenshots


<img width="1318" alt="Screenshot 2024-04-25 at 3 17 43 PM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/ca269c51-531f-4a48-a6dd-cf8ad16c6de8">
<img width="615" alt="Screenshot 2024-04-25 at 3 17 53 PM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/1a3d8837-535d-4344-b771-e3537bff5fdf">
<img width="423" alt="Screenshot 2024-04-25 at 3 18 00 PM" src="https://github.com/vigetlabs/wordpress-site-starter/assets/91974372/9df8e532-a849-446a-86b7-7e16cf120ca5">
